extensions/landing-zone/main.tf

```diff
@@ -0,0 +1,25 @@
+##############################################################################
+# IBM WebSphere Liberty operator deployment on the OCP cluster
+##############################################################################
+
+data "ibm_container_cluster_config" "cluster_config" {
+  cluster_name_id = var.cluster_id
+  config_dir      = "../../kubeconfig"                                                                      # See https://github.ibm.com/GoldenEye/issues/issues/552
```

Comment:
I would not include a `config_dir` for this DA as it will run in its own runtime and not clash with anything else